torchbenchmark/models/timm_resnest/__init__.py

```diff
@@ -58,19 +64,21 @@ def set_eval(self):
         pass
 
     def _step_eval(self):
-        output = self.eval_model(self.cfg.infer_example_inputs)
+        output = self.eval_model(self.infer_example_inputs)
 
     def get_module(self):
-        return self.model, (self.cfg.example_inputs,)
+        self.example_inputs = self.example_inputs.to(device=self.device)
```

Comment:
Should we preload for resnest too?

Reply:
> Should we preload for resnest too?

I don't think it matters as get_module is called outside of timed region.

Reply:
Okay,  how about for train and eval below?